mitiq/zne/tests/test_inference.py

```diff
@@ -275,7 +275,7 @@ def test_run_sequential_and_batched(factory, batched):
     # Compute expectation values at all the scale factors
     if batched:
 
-        def executor(circuits, kwargs) -> List[float]:
+        def executor(circuits, **kwargs) -> List[float]:
```

Comment:
Since this is a batched executor, the argument `kwargs` is a list of keyword arguments, so the signature should be 

```python
def executor(circuits, kwargs) -> List[float]
```

Maybe `kwargs` can be renamed to make this clearer.

Reply:
In the last commit I just removed the extra argument.
Note: I tried to write the code such that it is possible to define a batched executor without the `kwargs_list` argument, since this is "used" by a factory only in the particular case in which `shot_list` is used.